docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     build: ./
     working_dir: /home/node/uwazi
     ports:
-      - 3000:3000
+      - 127.0.0.1:3000:3000
```

Comment:
Can you (for this pull request) leave this line like it was before? Not that this additional change would not be accepted, but for make the other change be accepted as soon as possible, and we make one pull request per feature.

The change of the exposed port of the Uwazi itself have more side effects than make both Elastic Search and MongoDB only accessible via docker network